x-pack/plugin/sql/qa/server/src/main/resources/datetime.csv-spec

```diff
@@ -131,6 +131,32 @@ SELECT WEEK(birth_date) week, birth_date FROM test_emp ORDER BY WEEK(birth_date)
 44             |1961-11-02T00:00:00.000Z
 ;
 
+isoWeekOfYear
+schema::birth_date:ts|iso_week:i|week:i
+SELECT birth_date, IW(birth_date) iso_week, WEEK(birth_date) week FROM test_emp WHERE IW(birth_date) < 8 ORDER BY iso_week;
```

Comment:
Minor: maybe you can add a `AND week > 2` so we can check both filters work and one filter uses also the alias`